source/jsonnet/northern-ireland/household/blocks/who-lives-here/who_lives_here_interstitial.jsonnet

```diff
@@ -21,8 +27,7 @@ local contentDescription = {
         title: 'You will need to know',
         list: [
           'Names of the people living at this address including anyone temporarily away or who has been or intends to be in the UK for 3 months or more.',
-          'Names of visitors staying overnight at this address on 13 October 2019',
-          // This date is currently hard coded as validator currently does not allow placeholders within list object.
+          optionDescription,
```

Comment:
The variable name is misleading. We should consider not using a variable and inlining the placeholder Jsonnet here.